ipyopenlayers/Map.py

```diff
@@ -39,6 +39,10 @@ class GeoJSON(Layer):
     style = Dict({}).tag(sync=True)
     visible = Bool(True).tag(sync=True)
 
+class GeoTIFFTileLayer(Layer):
+   _model_name = Unicode('GeoTIFFTileLayerModel').tag(sync=True)
+   _view_name = Unicode('GeoTIFFTileLayerView').tag(sync=True)
+   sources = List(Dict).tag(sync=True)
```

Comment:
Because you're only reading the "url" property from the sources, let's make it simple for the user and replace this with:

```python
url = Unicode().tag(sync=True)
```

Otherwise, unless we have proper documentation, the user won't know what to set for "sources". With just "url" it's clear that they need to pass a url to a GeoTIFF file